docs/src/main/paradox/running-the-application.md

```diff
@@ -0,0 +1,86 @@
+Running the application
+-----------------------
+
+You can run the Hello World application from the command line or an IDE. The final topic in this guide describes how to run it from IntelliJ IDEA. However, before we get there, let’s take a quick look at the build tool: sbt.
+
+## The build files
+
+sbt uses a build.sbt file to handle the project. This project’s build.sbt file looks like this:
+
+@@snip [build.sbt]($g8root$/build.sbt)
+
+## Running the project
+
+We run the application from a console/terminal window:
+
+1. Enter `./sbt` on OSX/Linux or `sbt.bat` on Windows
+
+sbt downloads project dependencies. The `>` prompt indicates sbt has started in interactive mode.
+
+2. At the sbt prompt, enter `run`.
+
+The output should look like this:
+
+```
+...
+[info] Running com.lightbend.akka.http.sample.QuickstartServer
+Server online at http://localhost:8080/
+Press RETURN to stop...
+```
+
+## Testing the application
```

Comment:
I think we should show testing - the http testkit, and this section can remain as something next to it.
Let me know if you need help on that, here's the docs: http://doc.akka.io/docs/akka-http/current/scala/http/routing-dsl/testkit.html

it also allows testing without actually starting the entire web server which allows for very fast tests :)